test/micromamba/scenarios.json

```diff
@@ -3,7 +3,7 @@
         "image": "debian:stable-slim",
         "features": {
             "micromamba": {
-                "version": "0"
+                "version": "0.27.0"
```

Comment:
This means that you will need to edit here each time a new version is published. Do you want to do that?